src/main/java/com/yworks/common/ant/EntryPointsSection.java

```diff
@@ -12,43 +12,88 @@
  */
 public class EntryPointsSection extends Exclude {
 
-  private List<MethodSection> methodSections = new ArrayList<MethodSection>( 5 );
-  private List<ClassSection> classSections = new ArrayList<ClassSection>( 5 );
-  private List<FieldSection> fieldSections = new ArrayList<FieldSection>( 5 );
-  private List<AttributesSection> attributesSections = new ArrayList<AttributesSection>( 1 );
+  private final List<MethodSection> methodSections = new ArrayList<MethodSection>(5);
```

Comment:
Are these fixes applied automagically by your IDE?

Reply:
Looks like an incompatible code reformatting to me. While it seems that they don't change the code, I would rather have the PR not affect that many source lines of code because of merely a code reformatting.

Reply:
@yGuy even worse. Looks like `final` has been inserted here. Static code analysis bundled with IntellIJ does this, but it only takes you so far. Needs manual review, and I agree, changing this many LOC is simply unlucrative. Maybe we should ditch #77 altogether and simply ignore warnings.

Reply:
There should be a way to only get the javadoc improvements. That would make it far easier to review and verify that we don't get any other unwanted changes in the code. Static code analysis *automatically* inserts the `final`s? You can definitely turn that off if you don't want that - just like the simpler reformatting.